utilities.tex

```diff
@@ -563,7 +596,37 @@
 For templates \tcode{greater}, \tcode{less}, \tcode{greater_equal}, and
 \tcode{less_equal}, the specializations for any pointer type yield a total order,
 even if the built-in operators \tcode{<}, \tcode{>}, \tcode{<=}, \tcode{>=}
-do not.
+do not. \ednote{The following sentence is taken from the proposed resolution of
+\href{https://cplusplus.github.io/LWG/lwg-active.html\#2450}{LWG \#2450}.}
+\newtxt{For template specializations \tcode{greater<void>}, \tcode{less<void>},
+\tcode{greater_equal<void>}, and \tcode{less_equal<void>}, if the call operator calls a
+built-in operator comparing pointers, the call operator yields a total order.}
+
+\newtxt{
+\pnum
+Given a relational operator \tcode{\textit{OP}}, an object \tcode{x} of type \tcode{X},
+and an object \tcode{y} of type \tcode{Y}, let \tcode{\textit{BUILTIN_PTR_CMP}(X, \textit{OP}, Y)}
+be true if an only if \tcode{\textit{OP}} in the expression \tcode{(X\&\&)x \textit{OP} (Y\&\&)y} resolves
+to a built-in operator comparing pointers.
+}
+
+\newtxt{
+\pnum
+For all object types \tcode{T} for which there exists a specialization \tcode{less<T>},
+the instantiation \tcode{less<T>} shall satisfy \tcode{StrictWeakOrder<less<T>, T>()}~(\ref{concepts.lib.callables.strictweakorder}).
+In addition, \tcode{Relation<greater<T>, T>()}~(\ref{concepts.lib.callables.relation}), \tcode{Relation<greater_equal<T>, T>()},
+and \tcode{Relation<less_equal<T>, T>()} shall be satisfied and those class template
+instantiations shall be semantically consistent with \tcode{less<T>}~(\cxxref{operators}).
```

Comment:
`greater` is a `StrictWeakOrder` as well. We need to detail what "semantically consistent" means in this particular case instead of trying to define "semantically consistent": I fear the phrase is too hot and will burn us much like "semantically equivalent" did. I would rather require the specializations to be consistent only if they are valid, so that a user can e.g. define only `less<Widget>` if `greater<Widget>` et al are ill-formed.